src/components/LeaderboardTable.js

```diff
@@ -6,6 +6,14 @@ import LeaderboardHandle from "./LeaderboardHandle";
 const LeaderboardTable = ({ results }) => {
   const columns = React.useMemo(
     () => [
+      {
+        Header: 'Rank',
+        Cell: (props) => {
+          return (
+            <p>{props.row.original.rank}.</p>
+          );
+        }
+      },
```

Comment:
Here's simple row numbering.

```suggestion
      {
        Header: '#',
        Cell: (props) => {
          return (
            <p>{props.flatRows.indexOf(props.row) + 1}</p>
          );
        }
      },
```

Reply:
Clever!